src/main/java/sigma/internship/petProject/entity/Game.java

```diff
@@ -0,0 +1,46 @@
+package sigma.internship.petProject.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity(name = "game")
+@Table(name = "game")
+@Getter
+@Setter
+@ToString
```

Comment:
It's better to use shortcut for annotations ToString, EqualsAndHashCode, Getter on all fields, Setter on all non-final fields, and RequiredArgsConstructor - `@Data` from a Lombok.
For more information: https://projectlombok.org/features/Data

Please, use `@Data` annotation in all corresponding cases.